doc/install/index.rst

```diff
@@ -11,15 +13,17 @@ For users already familiar with Python:
 
       $ pip install mne  # dependencies are numpy, scipy, matplotlib
```

Comment:
Technically, the only hard dependencies are `numpy` and `scipy`; `matplotlib` doesn't get installed automatically with `pip install mne`.